.github/workflows/pylint.yml

```diff
@@ -0,0 +1,34 @@
+name: Pylint
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+    - name: Get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v11.5
+    - name: Analysing the code with pylint
+      run: |
+        PYTHONFILES=""
+        for file in ${{ steps.changed-files.outputs.all_modified_files }}; do
+          if [[ -f $file && $file == *.py ]]
+            then
+              PYTHONFILES="${PYTHONFILES} $file"
+              echo $PYTHONFILES
+              pylint $PYTHONFILES
```

Comment:
Does this run pylint on the current file as well as previously seen files? I think it'd make more sense to just do `pylint $file`

Reply:
In this case you wouldn't need PYTHONFILES either, and you can change the echo to just echo the current file as well

Reply:
Thanks for letting me know! I'll change that then